src/Pact/Types/Term/Internal.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
```

Comment:
Is this needed? It's the only non-whitespace change in this file...

Reply:
Deleted